core/templates/base.html

```diff
@@ -293,7 +294,7 @@ <h3>{{ site.title | safe }}</h3>
 
 		<script src="{{ STATIC_URL }}3rd/jquery/1.10.2/jquery.min.js"></script>
 		<script src="{{ STATIC_URL }}3rd/moment.js/2.10.6/moment.min.js"></script>
-		<script src="{{ STATIC_URL }}3rd/prettify/r298/prettify.js"></script>
+		<script src="{% static "3rd/prettify/r298/prettify.js" %}"></script>
```

Comment:
ideally we would use this new syntax for all assets, but for now there's complications & it works without it.